### PR TITLE
MS teams integration

### PR DESCRIPTION
**Pull request by janet-can**

### CODE REVIEW

File: soda/integrate-msteams.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* You have a Soda Cloud account with [Admin permissions]({% link soda-cloud/roles-and-rights.md %}).
+* You have [connected your Soda Cloud account]({% link soda-core/connect-core-to-cloud.md %}) to Soda Core and run at least one scan so as to produce one or more check results.<br /> OR<br /> You have [added a datasource]({% link soda-cloud/add-datasource.md %}) via a [Soda Agent]({% link soda-agent/deploy.md %}) and are ready to create an [agreement]({% link soda-cloud/agreements.md %})![preview](/assets/images/preview.png){:height="60px" width="60px"}.
+
+
+## Configure an MS Teams integration
+
+1. In your Soda Cloud account, navigate to **your avatar** > **Organization Settings**, then select the **Integrations** tab.
+2. Click the **+** at the upper right of the table of integrations to add a new integration. 
+3. In the **Add Integration** dialog box, select **Microsoft Teams**.
+4. In the first step of the guided integration workflow, download the Soda icon that you need to set up an incoming webhook with MS Tems.
```
Comment (dirkgroenen) [inline soda/integrate-msteams.md:27]: Note the type `Teams` 👼

File: soda/integrate-msteams.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+1. In your Soda Cloud account, navigate to **your avatar** > **Organization Settings**, then select the **Integrations** tab.
+2. Click the **+** at the upper right of the table of integrations to add a new integration. 
+3. In the **Add Integration** dialog box, select **Microsoft Teams**.
+4. In the first step of the guided integration workflow, download the Soda icon that you need to set up an incoming webhook with MS Tems. 
+5. Navigate to your MS Teams account to set up an incoming webhook; see detailed instructions below.
+5. Returning to Soda Cloud with the URL for the incoming webhook, continue to follow the guided steps to complete the integration. Reference the following tables for guidance on the values to input in the guided steps. <br /> 
+
+| Field or Label  |  Guidance |
+| --------------- |  -------- |
+| Name | Provide a unique name for your webhook in Soda Cloud. **Required** |
```
Comment (dirkgroenen) [inline soda/integrate-msteams.md:33]: `your webhook` should be `your integration`?

File: soda/integrate-msteams.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+| URL | Input the incoming webhook URL you obtained from MS Teams. See [section below](#set-up-incoming-webhook-in-ms-teams) for details. **Required**  |
+| Enable to send notifications to this webhook when a check result triggers an alert. | Check to allow users to select this webhook as a destination for alert notifications when check results warn or fail. |
```
Comment (dirkgroenen) [inline soda/integrate-msteams.md:35]: `Enable to send notifications to Microsoft Teams when a check result triggers an alert.` is how the phrase is in our app

File: soda/integrate-msteams.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+| Use this webhook as the default notification channel for all check result alerts. | Check to automatically configure check results alert notifications to this webhook by default. <br />Users can deselect the webhook as the notification destination in an individual check, but it is the prepopulated destination by default.   |
```
Comment (dirkgroenen) [inline soda/integrate-msteams.md:36]: `Use Microsoft Teams as the default notification channel for all check result alerts.`

File: soda/integrate-msteams.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+| Enable to use this webhook to track and resolve incidents in Soda Cloud. | Check to allow users to send incident information to a destination. <br />For example, a user creating a new incident can choose to use this webhook to create a new issue in Jira.|
```
Comment (dirkgroenen) [inline soda/integrate-msteams.md:37]: `Enable to use use Microsoft Teams to track and resolve incidents in Soda Cloud.`

File: soda/integrate-msteams.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+| Channel URL |   |
```
Comment (dirkgroenen) [inline soda/integrate-msteams.md:38]: `required` only when the `incidents` setting above is enabled. The team channel link can be obtained [in teams] by right clicking on the channel name in the overview sidebar.

**Review by dirkgroenen — COMMENTED** [review]
It might have been too early, but for what its worth I already went through the doc and left some comments here. Feel free to ignore the ones you were already aware of 🙂

File: soda/integrate-msteams.md (hunk 3 above)
Comment (janet-can) [reply soda/integrate-msteams.md:35]: Cool. Got it.

File: soda/integrate-msteams.md (hunk 6 above)
Comment (janet-can) [reply soda/integrate-msteams.md]: Great, I filled in that blank with more info of the kind.

**Pull request merged**